sci-biology/mne-python/mne-python-0.11.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Python package for MEG and EEG data analysis"
+HOMEPAGE="http://martinos.org/mne/mne-python.html"
+SRC_URI="https://github.com/mne-tools/mne-python/archive/v${PV}.zip -> ${P}.zip"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="test"
+#IUSE="cuda test"
+KEYWORDS="~x86 ~amd64"
+
+RDEPEND="
+	dev-python/numpy[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]
+	sci-libs/scikits_learn[${PYTHON_USEDEP}]
+	dev-python/joblib[${PYTHON_USEDEP}]
+	sci-libs/nibabel[${PYTHON_USEDEP}]
+	sci-biology/pysurfer[${PYTHON_USEDEP}]
+	dev-python/matplotlib[${PYTHON_USEDEP}]"
+
+#	cuda? (
+#		dev-python/pycuda[${PYTHON_USEDEP}]
+#		dev-python/scikits-cuda[${PYTHON_USEDEP}]
+#	)
+#"
+
+DEPEND="
+	test? ( dev-python/nose ${RDEPEND} )
+"
+
+S="${WORKDIR}/mne-python-${PV}"
+
+run_test() {
+	PYTHONPATH=. MNE_SKIP_SAMPLE_DATASET_TESTS=1 nosetests -v mne
+}
+
+python_test() {
+	distutils_install_for_testing
+	# Link to test data that won't be included in the final installation
+	local TEST_DIR="${T}/test-${EPYTHON}/$(python_get_sitedir)"
+	cd "${S}" || die
+	find . -type d -name data -exec ln -s "${S}"/{} ${TEST_DIR}/{} \; || die
+	cd ${TEST_DIR} || die
+	VIRTUALX_COMMAND="run_test"
+	virtx
```

Comment:
hmmmm.... ok, so I replace the last two lines with `virtx "run_test"`? qgrep doesn't show me many examples on this,,,